### PR TITLE
Use http-relative-path for wrapper_server

### PR DESCRIPTION
**Pull request by nblock**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -224,6 +226,7 @@
   Stdlib::IP::Address $http_host = '0.0.0.0',
   Stdlib::Port $http_port = 8080,
   Stdlib::Port $https_port = 8443,
+  String $http_relative_path = '/',
```
Comment (treydock) [inline manifests/init.pp:229]:
```suggestion
  Stdlib::Unixpath $http_relative_path = '/',
```
While this isn't a Unix path, the fact this must start with "/" would have same validation behavior.

**Comment by treydock** [comment]
Will need to update unit tests here:

https://github.com/treydock/puppet-module-keycloak/blob/b61f3f6e855cc4f611c7786b3804bc2c5dfba0b3/spec/classes/init_spec.rb#L125-L137

File: manifests/init.pp [hunk 2]
```diff
@@ -224,6 +226,7 @@
   Stdlib::IP::Address $http_host = '0.0.0.0',
   Stdlib::Port $http_port = 8080,
   Stdlib::Port $https_port = 8443,
+  Regexp[/^\/.*/] $http_relative_path = '/',
```
Comment (treydock) [inline manifests/init.pp:229]:
```suggestion
  Pattern[/^\/.*/] $http_relative_path = '/',
```

**Comment by treydock** [comment]
Also need to merge master branch into this pull request to fix Ubuntu 20.04 acceptance tests.

**Comment by nblock** [comment]
> Also need to merge master branch into this pull request to fix Ubuntu 20.04 acceptance tests.

Updated test, rebased and pushed.

**Comment by treydock** [comment]
This will be released as v8.2.0 once Github Actions complete

**Pull request merged**